hbw/app/javascript/packs/hbw/components/claiming/menu_button.jsx

```diff
@@ -53,15 +55,18 @@ class HBWClaimingMenuButton extends Component {
     };
 
     return (
-      <div className="claiming-menu-button-container" ref={this.rootDiv}>
-        <div className="claiming-menu-button" onClick={this.handlePopUp}>
-          <HydraIcon />
-          <span className="claiming-count">
-            {myTasksCount}/{unassignedTasksCount}
-          </span>
+      <>
+        <div className="claiming-menu-button-container" ref={this.rootDiv}>
+          <div className="claiming-menu-button" onClick={this.handlePopUp}>
+            <HydraIcon />
+            <span className="claiming-count">
+              {myTasksCount}/{unassignedTasksCount}
+            </span>
+          </div>
+          {isOpen && <PopUp perPage={10} env={this.props.env} count={count} />}
         </div>
-        {isOpen && <PopUp perPage={10} env={this.props.env} count={count} />}
-      </div>
+        <ClaimingNotifier perPage={400} env={env} />
```

Comment:
i didn't get where does this option come from? 
perPage

Reply:
this come from HOC wrapper `withClaimingTasks`